libs/librepcb/projecteditor/schematiceditor/symbolinstancepropertiesdialog.ui

```diff
@@ -186,6 +186,9 @@
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_9">
+          <property name="enabled">
+           <bool>false</bool>
+          </property>
```

Comment:
Could you maybe just completely remove the this label and its corresponding checkbox? I think it may confuse users if there is a disabled checkbox. And adding it again later is quite easy...